plugins/lime-plugin-pirania/src/piraniaEpics.js

```diff
@@ -0,0 +1,26 @@
+import { getGovernance } from './piraniaApi';
+import {
+	LOAD_GOVERNANCE,
+	LOAD_GOVERNANCE_SUCCESS,
+	LOAD_GOVERNANCE_ERROR,
+} from './piraniaConstants';
+
+import { Observable } from 'rxjs/Observable';
+import 'rxjs/add/operator/map';
+import 'rxjs/add/operator/catch';
+import 'rxjs/add/operator/concatMap';
+import 'rxjs/add/operator/concatAll';
+import 'rxjs/add/operator/mergeMap';
+
+const loadGovernance = ( action$, store, { wsAPI }) =>
+	action$.ofType(LOAD_GOVERNANCE)
+		.mergeMap((action) => getGovernance(wsAPI,store.value.meta.sid, {}))
+		.mergeMap(payload => {
+			if (payload.code) {
+				return [{ type: LOAD_GOVERNANCE_ERROR, payload }]; 
+			}
+			return [{ type: LOAD_GOVERNANCE_SUCCESS, payload }];
+		})
+		.catch([({ type: LOAD_GOVERNANCE_ERROR })]);
+
+export default { loadGovernance };
```

Comment:
Add line return at the end of each file (so when it changes, only one line changes, not the one that gets the newline character).